packages/seed/src/cli.ts

```diff
@@ -342,6 +345,64 @@ function addRegisterCommands(cli: Argv) {
     });
 }
 
+function addValidateCommands(cli: Argv) {
+    cli.command("validate", "Validate your changelog file", (yargs) => {
+        yargs
+            .command(
+                "cli",
+                "validate CLI releases",
+                (addtlYargs) =>
+                    addtlYargs.option("log-level", {
+                        default: LogLevel.Info,
+                        choices: LOG_LEVELS
+                    }),
+                async (argv) => {
+                    const taskContextFactory = new TaskContextFactory(argv["log-level"]);
+                    const context = taskContextFactory.create("Register");
+
+                    await validateCliRelease({
+                        context
+                    });
+                }
+            )
+            .command(
+                "generator <generator>",
```

Comment:
Do you imagine we'll want an option that can validate _all_ generators? I suppose we can always make the positional parameter optional and roll out a `--all` flag (or somethimg similar).